### PR TITLE
Date

### PR DESCRIPTION
**Pull request by Angelica137**
add mock date to test date stamp when Transaction is generated.

### CODE REVIEW

File: spec/account.test.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { moment } from "moment";
 import { sinon } from "sinon";
 var assert = require("assert");
 
-describe("Account", () => {
+xdescribe("Account", () => {
```
Comment (Edward-Phillips) [inline spec/account.test.js:7]: pended tests🤔

File: spec/transaction.test.js [hunk 2]
```diff
@@ -22,8 +23,11 @@ describe("Transaction", () => {
   //use mock to test date
   describe("#date", () => {
     it("stores the date when the deposit was made", () => {
-      let transaction = new Transaction(5.16);
+      const date = "Mon Mar 30 2020 00:00:00 GMT+0100 (British Summer Time)";
+      set(date);
```
Comment (Edward-Phillips) [inline spec/transaction.test.js:27]: nifty, very cool mocking library!

Comment (Steven-Klavins) [reply spec/transaction.test.js:27]: Moment is the same one I used it is indeed nifty! LGTM :)

Comment (grahamfalconer) [reply spec/transaction.test.js:27]: I like this a lot!

**Pull request merged**